### DISCUSSION TITLE
I am unable to login through github authenticator app

### DISCUSSION THREAD

**Discussion by AbhishekJupalli-1409** [community/community#135037] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

5 months ago i have restrated my mobile. Then i have lost my authenticator app for github. SO when i tr to login my github account it is asking for authentication code. So i have removed my email associated with that account and created a new account. But i need that old account . Does anyone know how to login in into it without email and authenticator app. I know the account name and password too..

**Comment by ahmedembeddedxx** [comment]
If you had set up 2FA, you might have a file with security codes to log in during emergencies. You can use these codes to access your account. If that doesn't help, try opening a support ticket and submitting it on behalf of someone else for your account.

Keep in mind, creating a new account with the same email reduces the chances of retrieving your old account. However, if you have the account name, password, and 2FA security codes, you should be able to access it.

**Reply by AbhishekJupalli-1409** [reply]
Thank you bro  @ahmedembeddedxx I'll try it once. But i dont have 2FA security codes. How to open a support ticket?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @AbhishekJupalli-1409 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by AbhishekJupalli-1409** [reply]
Thank you @queenofcorgis

**Comment by everforyou** [comment]
Hi,

To recover your GitHub account without your email and authenticator app:
Please check this github guidance.
https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

If it is not work too, I think you should contact GitHub Support for assistance.

**Reply by AbhishekJupalli-1409** [reply]
@BrotherRabbit91  Bro Actually i lost my authenticator app. And for my email, I have logged it out of my old acc and created a new acc with that mail. I don't Even have 2 factor recovery code.. 

![Screenshot (520)](https://github.com/user-attachments/assets/f577d516-3523-4304-8a34-e61a858b2188)

Can any anyone from github team can help me?

**Comment by everforyou** [comment]
If you save recover backup code,  you can also use it.

**Comment by AbhishekJupalli-1409** [comment]
Can't we just login normally without any of this authenticator, recovery codes and so and so??

**Reply by everforyou** [reply]
Check this.
https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

**Comment by everforyou** [comment]
There are all available method to login if  you lose your 2FA credentials, Okay?
